### ISSUE TITLE
cocoapods

### ISSUE THREAD

**Issue by 4fthawaiian** [vincentsaluzzo/Homebrew-GUI#3]
I don't see a mention of cocoapods as a req, but it definitely seems like you need it. No?

**Comment by vincentsaluzzo** [comment]
Yes we need to have a better presentation and we need to list all OS project we use (Cocoapods, ReactiveCocoa, etc.)
